### PR TITLE
Use unicode/bytes literals instead of calls

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
oh this is a superset of #4305

**Comment by asottile** [comment]
@RonnyPfannschmidt more a continuation -- I was inspired by your change to try and fix more of it :)

File: src/_pytest/assertion/util.py [hunk 1]
```diff
@@ -262,16 +254,16 @@ def _compare_eq_sequence(left, right, verbose=False):
     explanation = []
     for i in range(min(len(left), len(right))):
         if left[i] != right[i]:
-            explanation += [u("At index %s diff: %r != %r") % (i, left[i], right[i])]
+            explanation += [u"At index %s diff: %r != %r" % (i, left[i], right[i])]
```
Comment (RonnyPfannschmidt) [inline src/_pytest/assertion/util.py]: at the diff i noted that we should use saferepr here, perhaps a  nice followup

**Comment by RonnyPfannschmidt** [comment]
great job at pulling in the rest :+1:

**Pull request merged**